sdk/digitaltwins/azure-digitaltwins-core/src/samples/java/com/azure/digitaltwins/core/PublishTelemetrySyncSamples.java

```diff
@@ -0,0 +1,135 @@
+    }
+
+    public static void runPublishTelemetrySample() throws JsonProcessingException {
+
+        ConsoleLogger.printHeader("Telemetry Samples");
+
+        // For the purpose of this example we will create temporary models using a random model Ids.
+        // We have to make sure these model Ids are unique within the DT instance.
+
+        String componentModelId = UniqueIdHelper.getUniqueModelId(SamplesConstants.TEMPORARY_COMPONENT_MODEL_PREFIX, client, randomIntegerStringGenerator);
+        String modelId = UniqueIdHelper.getUniqueModelId(SamplesConstants.TEMPORARY_MODEL_PREFIX, client, randomIntegerStringGenerator);
+        String digitalTwinId = UniqueIdHelper.getUniqueDigitalTwinId(SamplesConstants.TEMPORARY_TWIN_PREFIX, client, randomIntegerStringGenerator);
+
+        String newComponentModelPayload = SamplesConstants.TEMPORARY_COMPONENT_MODEL_PAYLOAD
+            .replace(SamplesConstants.COMPONENT_ID, componentModelId);
+
+        String newModelPayload = SamplesConstants.TEMPORARY_MODEL_WITH_COMPONENT_PAYLOAD
+            .replace(SamplesConstants.MODEL_ID, modelId)
+            .replace(SamplesConstants.COMPONENT_ID, componentModelId);
+
+        List<String> modelsList = new ArrayList<>(Arrays.asList(newComponentModelPayload, newModelPayload));
+
+        ConsoleLogger.printHeader("Create Models");
+        // We now create all the models (including components)
+        List<ModelData> modelList =  client.createModels(modelsList);
+
+        for (ModelData model : modelList) {
+            ConsoleLogger.print("Created model: " + model.getId());
+        }
+
+        ConsoleLogger.printHeader("Create DigitalTwin");
+
+        String twinPayload = SamplesConstants.TEMPORARY_TWIN_PAYLOAD
+            .replace(SamplesConstants.MODEL_ID, modelId);
+
+        client.createDigitalTwin(digitalTwinId, twinPayload);
+
+        ConsoleLogger.printSuccess("Created digital twin " + digitalTwinId);
+        try
+        {
+            ConsoleLogger.printHeader("Publish Telemetry");
+            // construct your json telemetry payload by hand.
+            client.publishTelemetry(digitalTwinId, "{\"Telemetry1\": 5}");
+            ConsoleLogger.print("Published telemetry message to twin " + digitalTwinId);
+
+            ConsoleLogger.printHeader("Publish Component Telemetry");
+            PublishTelemetryRequestOptions componentTelemetryRequestOptions = new PublishTelemetryRequestOptions();
+
+            // construct your json telemetry payload using a hashtable.
+            Dictionary<String, Integer> telemetryPayload = new Hashtable<>();
+            telemetryPayload.put("ComponentTelemetry1", 9);
+            String telemetryStringPayload = new ObjectMapper().writeValueAsString(telemetryPayload);
+
+            Response<Void> publishComponentTelemetryResponse = client.publishComponentTelemetryWithResponse(
+                digitalTwinId,
+                "Component1",
+                telemetryStringPayload,
```

Comment:
Did we remove the extra deserialization to Object that was happening in the CL for these APIs?

Reply:
I did in the Telemetry test PR